Sources/Protocols/MessagesDataSource.swift

```diff
@@ -81,6 +81,14 @@ public protocol MessagesDataSource: AnyObject {
     /// The default value returned by this method is `nil`.
     func cellBottomLabelAttributedText(for message: MessageType, at indexPath: IndexPath) -> NSAttributedString?
 
+    /// Additional config for a cell, called in cellForItem method. e.g. customize accessoryView.   
+    ///
+    /// - Parameters:
+    ///   - cell: The `MessageCollectionViewCell` that can be configured.
+    ///   - indexPath: The `IndexPath` of the cell.
+    ///
+    /// The default value returned by this method is `nil`.
+    func additionalConfig(for cell: MessageCollectionViewCell, at indexPath: IndexPath)
```

Comment:
I'm looking into this, but we may not want to add it in this PR. Let's keep the scope small

Reply:
What's your suggestion on letting clients config the accessoryView? A specific setupAccessoryView delegate?